libkbfs/bare_root_metadata.go

```diff
@@ -271,6 +271,12 @@ func (md *BareRootMetadataV2) DeepCopy(codec Codec) (BareRootMetadata, error) {
 	return &newMd, nil
 }
 
+// MakeSuccessor implements the ImmutableBareRootMetadata interface for BareRootMetadataV2.
+func (md *BareRootMetadataV2) MakeSuccessor(codec Codec) (BareRootMetadata, error) {
+	// MDv3 TODO: Make a v3 successor.
+	return md.DeepCopy(codec)
```

Comment:
That `MakeSuccessor` shells out to a helper method to make a suitable copy. It seemed logical to do the conversion here in a versioned implementation method.

>Also, don't we still need to continue making V2 successors, at least for a while?

At least until everything else in the server end is caught up, yes.